@here/harp-examples/src/tile_dependencies.ts

```diff
@@ -0,0 +1,288 @@
+/*
```

Comment:
I wonder if we should/could simplify this example a bit.
It might be enough to have a synchronous datasource that has tile dependencies, even though that would not cover the decode tile depency propagation part.

Reply:
I discussed this with @robertoraggi , he wasn't so keen to have an example that shows how to use the tile.dependencies property, because it could be copy / pasted by others (@robertoraggi , correct me if I misunderstood) and hence misused.

Hence adapting the example to use the dependencies in the `DecodedTile`.